std/math.d

```diff
@@ -5589,7 +5589,7 @@ real NaN(ulong payload) @trusted pure nothrow @nogc
     }
 }
 
-pure nothrow @nogc unittest
+@safe pure nothrow @nogc unittest
```

Comment:
Annoyingly, this had been fixed just three months before...
https://github.com/dlang/phobos/commit/fc7ab7fcae8bb4ff5ad3c8b40fe4b19a6b8654cf